### PR TITLE
CORE-8969 Added token selection flows to the UTXO demo app

### PR DESCRIPTION
**Pull request by owenstanford**
Added token selection flows to the UTXO demo app to aid manual testing

### CODE REVIEW

File: testing/cpbs/ledger-utxo-demo-app/src/main/kotlin/net/cordapp/demo/utxo/token/CreateCoinFlow.kt [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+                    participants = participants,
+                    tag = creationRequest.tag,
+                    ownerHash = ownerHash
+                )
+            }
+
+            val notary = notaryLookup.notaryServices.first()
+            val notaryKey = memberLookup.lookup().first {
+                it.memberProvidedContext["corda.notary.service.name"] == notary.name.toString()
+            }.ledgerKeys.first()
```
Comment (blsemo) [inline testing/cpbs/ledger-utxo-demo-app/src/main/kotlin/net/cordapp/demo/utxo/token/CreateCoinFlow.kt]: Shouldn't use `first` to get notaries/keys as per Ramzi's message.

File: testing/cpbs/ledger-utxo-demo-app/src/main/kotlin/net/cordapp/demo/utxo/token/SpendCoinFlow.kt [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    @CordaInject
+    lateinit var tokenSelection: TokenSelection
+
+    @Suspendable
+    override fun call(requestBody: RPCRequestData): String {
+        log.info("Starting Coin Spend...")
+        try {
+            val spendRequest = requestBody.getRequestBodyAs<SpendCoinMessage>(jsonMarshallingService)
+            val bankX500 = MemberX500Name.parse(spendRequest.issuerBankX500)
+            val notary = notaryLookup.notaryServices.first()
```
Comment (blsemo) [inline testing/cpbs/ledger-utxo-demo-app/src/main/kotlin/net/cordapp/demo/utxo/token/SpendCoinFlow.kt]: Again, don't use first

**Review by blsemo — DISMISSED** [review]
Looks good to me apart from the notary selection.

File: testing/cpbs/ledger-utxo-demo-app/src/main/kotlin/net/cordapp/demo/utxo/token/SpendCoinFlow.kt (hunk 2 above)
Comment (owenstanford) [reply testing/cpbs/ledger-utxo-demo-app/src/main/kotlin/net/cordapp/demo/utxo/token/SpendCoinFlow.kt]: fixed

File: testing/cpbs/ledger-utxo-demo-app/src/main/kotlin/net/cordapp/demo/utxo/token/CreateCoinFlow.kt (hunk 1 above)
Comment (owenstanford) [reply testing/cpbs/ledger-utxo-demo-app/src/main/kotlin/net/cordapp/demo/utxo/token/CreateCoinFlow.kt]: fixed

**Pull request closed without merging**